scripts/jams_to_mirex_pattern.py

```diff
@@ -0,0 +1,65 @@
+
+import csv
+import json
+
+x = open('/path/to/file.jams').read()
```

Comment:
A couple of points here:

1) It's better to use context managers for this, so that errors are handled properly.  Instead of

```python
x = open('file').read()
x = json.load(x)
```
do
```python
with open('file.jams', 'r') as fdesc:
    x = json.load(fdesc)
```

Though, really it's better to use `jams.load('file.jams')` so that the data structure is parsed into a JAMS object, rather than loading raw json.

2) Filenames should be given as command line arguments.  Check out the `jams_to_lab.py` script to see how we do this currently.

Reply:
Added argparse. 

Trying to load using jams.load but got a type error. Opened an issue. #196